backend/src/apis/shared/userinfo.ts

```diff
@@ -24,6 +31,7 @@ const userInfo = {
 			organization: org,
 			name: req.jwtClaims.name,
 			email: req.jwtClaims.email,
+			role: roleEnums[`${req.jwtClaims.roles[0]}`] || 'none',
```

Comment:
What if they have more than one?

Reply:
All good to go on this front. mapped the array to the enumerated values and used join() to display the roles.